recipe/magento2.php

```diff
@@ -19,15 +25,15 @@
 set('shared_dirs', [
     'var/composer_home',
     'var/log',
-    'var/cache',
     'var/export',
     'var/report',
+    'var/import',
     'var/import_history',
     'var/session',
     'var/importexport',
     'var/backups',
     'var/tmp',
-    'pub/sitemaps',
```

Comment:
Why did you change this?

Reply:
I think we should add 'pub/sitemap' (instead of replace) to keep backwards compatibility, or we should include this change in the next major release.

Reply:
It doesn't break BC in terms of deployment, but people's sitemaps could be gone if they've relied on the pub/sitemaps naming dictated by this recipe.

Reply:
We are actually about to land v7. So breaking BC is ok.

Reply:
Fine by me!